pkg/internal/patch/toml.go

```diff
@@ -64,6 +79,17 @@ func TOML(toPatch string, patches []string, patches6902 []string) (string, error
 	return jsonToTOMLString(j)
 }
 
+func containerdConfigVersion(configTOML string) (int, error) {
+	type version struct {
+		Version int `toml:"version,omitempty"`
+	}
+	v := version{}
+	if err := toml.Unmarshal([]byte(configTOML), &v); err != nil {
```

Comment:
> The version field in the config file specifies the config’s version. If no version number is specified inside the config file then it is assumed to be a version 1 config and parsed as such. Please use version = 2 to enable version 2 config as version 1 has been deprecated.

should we default to version 1 instead of error?

Reply:
version 1 isn't the default in containerd as far as I know, but the default would depend on containerd version and that seems fragile.

This is the file to be patched, we are always specifying the version ourselves, so if it doesn't have the version or we can't parse it, something is wrong.

since this is in an internal package already, I didn't think it was worth designing for other use cases (hence the rename to Containerd and the assumption about the `version` key)